sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -27,7 +27,7 @@
 int hda_dsp_ctrl_link_reset(struct snd_sof_dev *sdev, bool reset)
 {
 	unsigned long timeout;
-	u32 gctl = 0;
+	u32 gctl;
```

Comment:
theoretically this is wrong too - if the loop below isn't entered, gctl remains uninitialised and it's then printed out